lib/ssl/src/ssl.erl

```diff
@@ -25,20 +25,46 @@
 
 -module(ssl).
 -moduledoc """
-Interface Functions for Secure Socket Layer
+Interface Functions for TLS (Transport Layer Security),
+and DTLS (Datagram Transport Layer Security) are supported. The applications
+name is still ssl due to the fact that the first versions of the TLS protcol
```

Comment:
```suggestion
Interface functions for TLS (Transport Layer Security)
and DTLS (Datagram Transport Layer Security).

The applications name is still ssl due to the fact that the first versions of the TLS protcol
```

The first paragraph is a summary, that is shown [in the module index](https://www.erlang.org/docs/27/man_index.html) and other places, so best to keep it brief.

Also, I think it should be "Interface functions", nor "Interface Functions" as it is not a heading. And no comma before "and" if the listing only has two items.

Reply:
fixed